tests/types/otp-test.json

```diff
@@ -0,0 +1,116 @@
+{
+  "$schema": "https://packageurl.org/schemas/purl-test.schema-0.1.json",
+  "tests": [
+    {
+      "description": "Parse test for <class 'type'> PURL",
+      "test_group": "base",
+      "test_type": "parse",
+      "input": "pkg:otp/asn1@5.4.1?platform=linux&arch=amd64&repository_url=https:%2F%2Fgithub.com%2Ferlang%2Fotp&vcs_url=git%20https:%2F%2Fgithub.com%2Ferlang%2Fotp.git#src/asn1ct.erl",
+      "expected_output": {
+        "type": "otp",
+        "namespace": null,
+        "name": "asn1",
+        "version": "5.4.1",
+        "qualifiers": {
+          "platform": "linux",
+          "arch": "amd64"
+        },
```

Comment:
The parse results should include the decoded information for the `repository_url `and `vcs_url `qualifiers or fail due to the missing information in the parse results.